ql/src/test/results/clientpositive/llap/auto_join10.q.out

```diff
@@ -57,6 +57,7 @@ STAGE PLANS:
                 TableScan
                   alias: src
                   filterExpr: key is not null (type: boolean)
+                  probeDecodeDetails: cacheKey:HASH_MAP_MAPJOIN_30_container, bigKeyColName:key, smallTablePos:0, keyRatio:1.582
```

Comment:
why is `keyRatio` above 1? shouldn't it mean the expected selectivity of the operation?

Reply:
In general thats the case, this particular query though is a self join where tsKeyCardinality is 500 while mjKeyCardinality is 791 thus the ratio is above 1.